src/main/java/games/strategy/util/MD5Crypt.java

```diff
@@ -227,6 +219,22 @@ public static String crypt(final String password, String salt, final String magi
     return result.toString();
   }
 
+  /**
+   * Creates a new random salt that can be passed to {@link #crypt(String, String)}.
+   *
+   * @return A new random salt; never {@code null}.
+   */
+  public static String newSalt() {
```

Comment:
I extracted this method because it seemed kind of ridiculous to have to call

```java
MD5Crypt.getSalt(MD5Crypt.MAGIC, MD5Crypt.crypt("something"))
```

just to generate a new salt.